src/test/java/com/github/javafaker/ColorTest.java

```diff
@@ -11,4 +11,14 @@
     public void testName() {
         assertThat(faker.color().name(), matchesRegularExpression("(\\w+ ?){1,2}"));
     }
+
+    @Test
+    public void testHex() {
+        assertThat(faker.color().hex(), matchesRegularExpression("^#[0-9A-F]{6}$"));
+    }
+
+    @Test
+    public void testHexNoHashSign() {
```

Comment:
technically you should have a test that tests passing in a false... so `testHexWithHashSign()`